### PR TITLE
[BACKLOG-37669] - Focus outline and filename of focused file is partially visible - File Open Dialog

### PR DESCRIPTION
**Pull request by phalguni-a**
@pentaho/wcag , Please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 53s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-922/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 1]
```diff
@@ -5063,6 +5063,11 @@ div.dijitDateTextBox.dijitComboBox {
   width: 100%;
 }
 
+.fileChooserCellFileName:focus-visible {
+  outline: var(--focus-ring);
+  outline-offset: -2px;
+}
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:5069]: Apparently, all that is needed here is: `--focus-ring-offset: -2px;`.

**Review by dcleao — CHANGES_REQUESTED** [review]

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css (hunk 1 above)
Comment (phalguni-a) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:5069]: thanks for the suggestion. Addressing it.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-922/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Pull request merged**